yarn_api_client/resource_manager.py

```diff
@@ -146,13 +147,13 @@ def cluster_application_statistics(self, state_list=None,
         # TODO: validate state argument
         states = ','.join(state_list) if state_list is not None else None
         if application_type_list is not None:
-            applicationTypes = ','.join(application_type_list)
+            application_types = ','.join(application_type_list)
         else:
-            applicationTypes = None
+            application_types = None
 
         loc_args = (
             ('states', states),
-            ('applicationTypes', applicationTypes))
+            ('application_types', application_types))
```

Comment:
first item in tuple goes to hadoop api
does it expect `?construct_parameters=X` instead of `?applicationTypes=X` in query string?